### PR TITLE
fix(optimum): detect a compiled model dir by rbln_config.json

### PR DESCRIPTION
**Pull request by rebel-eunji**
### 🚀 Summary of Changes

`os.path.exists(model_path)` accepted any local directory, so a plain HuggingFace checkpoint -- which is what `model_path` points at under HF_HUB_OFFLINE=1 -- was taken for a pre-compiled model.

Gate on the top-level `rbln_config.json` instead. That is optimum-rbln's own marker for a compiled directory (`RBLNBaseModel._load_compiled_model_dir`), it always sits at the top level, and a plain HF checkpoint never has one.

Globbing for top-level `*.rbln` would not work: models whose config sets `_allow_no_compile_cfgs` (e.g. PaliGemma, which vLLM supports) compile no graph of their own and keep every `.rbln` inside submodule subdirectories.

Applying the same predicate to `cached_model_path` also stops a cache dir left half-written by an interrupted compile from counting as a hit.

---

### 📌 Related Issues / Tickets

* Resolves #
* Related to #

---

### ✅ Type of Change

* [ ] 🚀 Release (`release`)
* [ ] ✨ Feature (`feature`)
* [ ] 🧠 Model support (`model`)
* [ ] 🧬 Core engine changes (`core`)
* [x] 🛠 Bug fix (`fix`)
* [ ] ⚙️ Performance improvement (`perf`)
* [ ] 🔁 Refactor or code cleanup (`refactor`)
* [ ] 📄 Documentation (`docs`)
* [ ] ❓ Other (`other`): please describe

---

### 🧪 How to Test

1. Run `...`
2. Verify output: `...`
3. Edge case tested: `...`

---

### 📸 Screenshots / Logs (if applicable)

---

### 📋 Checklist

* [ ] PR title follows Conventional Commits format
* [ ] This PR is linked to an existing issue
* [ ] The test method is described, and the expected result is clearly stated
* [ ] Relevant documentation has been updated (if applicable)

---

### 💬 Notes

---

### CODE REVIEW

File: vllm_rbln/model_executor/models/optimum/model_base.py [hunk 1]
```diff
@@ -44,6 +44,12 @@
 logger = init_logger(__name__)
 
 
+def _is_compiled_dir(path: str | None) -> bool:
+    if path is None:
+        return False
+    return bool(path) and os.path.isfile(os.path.join(path, "rbln_config.json"))
```
Comment (rebel-seinpark) [inline vllm_rbln/model_executor/models/optimum/model_base.py]:
```suggestion
    if not path:
        return False
    return os.path.isfile(os.path.join(path, "rbln_config.json"))
```

Comment (rebel-eunji) [reply vllm_rbln/model_executor/models/optimum/model_base.py]: I applied it and refactored.

**Review by rebel-seinpark — APPROVED** [review]

**Review by rebel-cjlee — APPROVED** [review]
